 Network rinha-nginx-2024q1  Creating
 Network rinha-nginx-2024q1  Created
 Container bcaua-db-1  Creating
 Container bcaua-db-1  Created
 Container api02  Creating
 Container api01  Creating
 Container api01  Created
 Container api02  Created
 Container bcaua-nginx-1  Creating
 Container bcaua-nginx-1  Created
Attaching to api01, api02, db-1, nginx-1
db-1     | The files belonging to this database system will be owned by user "postgres".
db-1     | This user must also own the server process.
db-1     | 
db-1     | The database cluster will be initialized with locale "en_US.utf8".
db-1     | The default database encoding has accordingly been set to "UTF8".
db-1     | The default text search configuration will be set to "english".
db-1     | 
db-1     | Data page checksums are disabled.
db-1     | 
db-1     | fixing permissions on existing directory /var/lib/postgresql/data ... ok
db-1     | creating subdirectories ... ok
db-1     | selecting dynamic shared memory implementation ... posix
db-1     | selecting default max_connections ... 100
db-1     | selecting default shared_buffers ... 128MB
db-1     | selecting default time zone ... Etc/UTC
db-1     | creating configuration files ... ok
db-1     | running bootstrap script ... ok
api02    | warn: Microsoft.AspNetCore.Hosting.Diagnostics[15]
api02    |       Overriding HTTP_PORTS '8080' and HTTPS_PORTS ''. Binding to values defined by URLS instead 'http://+:3000'.
api02    | info: Microsoft.Hosting.Lifetime[14]
api02    |       Now listening on: http://[::]:3000
api02    | info: Microsoft.Hosting.Lifetime[0]
api02    |       Application started. Press Ctrl+C to shut down.
api02    | info: Microsoft.Hosting.Lifetime[0]
api02    |       Hosting environment: Production
api02    | info: Microsoft.Hosting.Lifetime[0]
api02    |       Content root path: /app
api01    | warn: Microsoft.AspNetCore.Hosting.Diagnostics[15]
api01    |       Overriding HTTP_PORTS '8080' and HTTPS_PORTS ''. Binding to values defined by URLS instead 'http://+:3000'.
api01    | info: Microsoft.Hosting.Lifetime[14]
api01    |       Now listening on: http://[::]:3000
api01    | info: Microsoft.Hosting.Lifetime[0]
api01    |       Application started. Press Ctrl+C to shut down.
api01    | info: Microsoft.Hosting.Lifetime[0]
api01    |       Hosting environment: Production
api01    | info: Microsoft.Hosting.Lifetime[0]
api01    |       Content root path: /app
db-1     | performing post-bootstrap initialization ... ok
nginx-1  | /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
nginx-1  | /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
nginx-1  | 10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
nginx-1  | 10-listen-on-ipv6-by-default.sh: info: Enabled listen on IPv6 in /etc/nginx/conf.d/default.conf
nginx-1  | /docker-entrypoint.sh: Sourcing /docker-entrypoint.d/15-local-resolvers.envsh
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
nginx-1  | /docker-entrypoint.sh: Configuration complete; ready for start up
db-1     | initdb: warning: enabling "trust" authentication for local connections
db-1     | initdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.
db-1     | syncing data to disk ... ok
db-1     | 
db-1     | 
db-1     | Success. You can now start the database server using:
db-1     | 
db-1     |     pg_ctl -D /var/lib/postgresql/data -l logfile start
db-1     | 
db-1     | waiting for server to start....2024-02-11 00:16:36.953 UTC [48] LOG:  starting PostgreSQL 16.1 (Debian 16.1-1.pgdg120+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
db-1     | 2024-02-11 00:16:36.962 UTC [48] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db-1     | 2024-02-11 00:16:36.994 UTC [51] LOG:  database system was shut down at 2024-02-11 00:16:36 UTC
db-1     | 2024-02-11 00:16:37.010 UTC [48] LOG:  database system is ready to accept connections
db-1     |  done
db-1     | server started
api01    | Failed to connect to 172.28.0.2:5432
db-1     | CREATE DATABASE
db-1     | 
db-1     | 
db-1     | /usr/local/bin/docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/postgres.conf
db-1     | 
db-1     | /usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/script.sql
db-1     | CREATE TABLE
db-1     | CREATE TABLE
db-1     | INSERT 0 5
db-1     | CREATE PROCEDURE
db-1     | 
db-1     | 
db-1     | 2024-02-11 00:16:37.387 UTC [48] LOG:  received fast shutdown request
db-1     | waiting for server to shut down....2024-02-11 00:16:37.391 UTC [48] LOG:  aborting any active transactions
db-1     | 2024-02-11 00:16:37.394 UTC [48] LOG:  background worker "logical replication launcher" (PID 54) exited with exit code 1
db-1     | 2024-02-11 00:16:37.396 UTC [49] LOG:  shutting down
db-1     | 2024-02-11 00:16:37.401 UTC [49] LOG:  checkpoint starting: shutdown immediate
db-1     | 2024-02-11 00:16:37.535 UTC [49] LOG:  checkpoint complete: wrote 940 buffers (5.7%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.094 s, sync=0.033 s, total=0.139 s; sync files=313, longest=0.003 s, average=0.001 s; distance=4296 kB, estimate=4296 kB; lsn=0/191CC90, redo lsn=0/191CC90
db-1     | 2024-02-11 00:16:37.543 UTC [48] LOG:  database system is shut down
db-1     |  done
db-1     | server stopped
db-1     | 
db-1     | PostgreSQL init process complete; ready for start up.
db-1     | 
db-1     | 2024-02-11 00:16:37.621 UTC [1] LOG:  starting PostgreSQL 16.1 (Debian 16.1-1.pgdg120+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
db-1     | 2024-02-11 00:16:37.621 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
db-1     | 2024-02-11 00:16:37.621 UTC [1] LOG:  listening on IPv6 address "::", port 5432
db-1     | 2024-02-11 00:16:37.626 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db-1     | 2024-02-11 00:16:37.633 UTC [66] LOG:  database system was shut down at 2024-02-11 00:16:37 UTC
db-1     | 2024-02-11 00:16:37.640 UTC [1] LOG:  database system is ready to accept connections
db-1     | 2024-02-11 00:18:04.794 UTC [164] FATAL:  sorry, too many clients already
db-1     | 2024-02-11 00:18:04.802 UTC [173] FATAL:  sorry, too many clients already
db-1     | 2024-02-11 00:18:04.803 UTC [174] FATAL:  sorry, too many clients already
api01    | 53300: sorry, too many clients already
api01    | 53300: sorry, too many clients already
api01    | 53300: sorry, too many clients already
db-1     | 2024-02-11 00:18:07.604 UTC [267] FATAL:  sorry, too many clients already
db-1     | 2024-02-11 00:18:07.604 UTC [269] FATAL:  sorry, too many clients already
db-1     | 2024-02-11 00:18:07.604 UTC [270] FATAL:  sorry, too many clients already
db-1     | 2024-02-11 00:18:07.604 UTC [271] FATAL:  sorry, too many clients already
db-1     | 2024-02-11 00:18:07.604 UTC [273] FATAL:  sorry, too many clients already
db-1     | 2024-02-11 00:18:07.605 UTC [274] FATAL:  sorry, too many clients already
db-1     | 2024-02-11 00:18:07.605 UTC [275] FATAL:  sorry, too many clients already
db-1     | 2024-02-11 00:18:07.605 UTC [276] FATAL:  sorry, too many clients already
api01    | fail: Microsoft.AspNetCore.Server.Kestrel[13]
api01    |       Connection id "0HN1AIQR4A40F", Request id "0HN1AIQR4A40F:00000001": An unhandled exception was thrown by the application.
api01    |       Npgsql.PostgresException (0x80004005): 53300: sorry, too many clients already
api01    |          at Npgsql.Internal.NpgsqlConnector.<ReadMessageLong>d__233.MoveNext() + 0x995
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Threading.Tasks.Sources.ManualResetValueTaskSourceCore`1.ThrowForFailedGetResult() + 0x16
api01    |          at System.Threading.Tasks.Sources.ManualResetValueTaskSourceCore`1.GetResult(Int16) + 0x3e
api01    |          at System.Runtime.CompilerServices.PoolingAsyncValueTaskMethodBuilder`1.StateMachineBox`1.System.Threading.Tasks.Sources.IValueTaskSource<TResult>.GetResult(Int16) + 0x2c
api01    |          at Npgsql.Internal.NpgsqlConnector.<Authenticate>d__0.MoveNext() + 0x1f6
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Npgsql.Internal.NpgsqlConnector.<<Open>g__OpenCore|213_1>d.MoveNext() + 0x531
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Npgsql.Internal.NpgsqlConnector.<Open>d__213.MoveNext() + 0x6d9
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Npgsql.PoolingDataSource.<OpenNewConnector>d__37.MoveNext() + 0x54b
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Npgsql.PoolingDataSource.<<Get>g__RentAsync|34_0>d.MoveNext() + 0x1c7
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Npgsql.NpgsqlConnection.<<Open>g__OpenAsync|42_0>d.MoveNext() + 0x3de
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at api.services.ClienteRepository.<CreateTransacao>d__3.MoveNext() + 0x14e
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at api.services.ClientesService.<Transacao>d__3.MoveNext() + 0x2d7
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Program.<>c.<<<Main>$>b__0_1>d.MoveNext() + 0x231
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Microsoft.AspNetCore.Http.Generated.<GeneratedRouteBuilderExtensions_g>FB49A76E00958A78ED8A3DC031D4149080B37ED6EB129C512F9112A3DB80F087D__GeneratedRouteBuilderExtensionsCore.<>c__DisplayClass3_0.<<MapPost0>g__RequestHandler|4>d.MoveNext() + 0x3b8
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.<ProcessRequests>d__238`1.MoveNext() + 0x46f
api01    |         Exception data:
api01    |           Severity: FATAL
api01    |           SqlState: 53300
api01    |           MessageText: sorry, too many clients already
api01    |           File: proc.c
api01    |           Line: 351
api01    |           Routine: InitProcess
api01    | fail: Microsoft.AspNetCore.Server.Kestrel[13]
api01    |       Connection id "0HN1AIQR4A40D", Request id "0HN1AIQR4A40D:00000001": An unhandled exception was thrown by the application.
api01    |       Npgsql.PostgresException (0x80004005): 53300: sorry, too many clients already
api01    |          at Npgsql.Internal.NpgsqlConnector.<ReadMessageLong>d__233.MoveNext() + 0x995
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Threading.Tasks.Sources.ManualResetValueTaskSourceCore`1.ThrowForFailedGetResult() + 0x16
api01    |          at System.Threading.Tasks.Sources.ManualResetValueTaskSourceCore`1.GetResult(Int16) + 0x3e
api01    |          at System.Runtime.CompilerServices.PoolingAsyncValueTaskMethodBuilder`1.StateMachineBox`1.System.Threading.Tasks.Sources.IValueTaskSource<TResult>.GetResult(Int16) + 0x2c
api01    |          at Npgsql.Internal.NpgsqlConnector.<Authenticate>d__0.MoveNext() + 0x1f6
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Npgsql.Internal.NpgsqlConnector.<<Open>g__OpenCore|213_1>d.MoveNext() + 0x531
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Npgsql.Internal.NpgsqlConnector.<Open>d__213.MoveNext() + 0x6d9
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Npgsql.PoolingDataSource.<OpenNewConnector>d__37.MoveNext() + 0x54b
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Npgsql.PoolingDataSource.<<Get>g__RentAsync|34_0>d.MoveNext() + 0x1c7
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Npgsql.NpgsqlConnection.<<Open>g__OpenAsync|42_0>d.MoveNext() + 0x3de
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at api.services.ClienteRepository.<CreateTransacao>d__3.MoveNext() + 0x14e
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at api.services.ClientesService.<Transacao>d__3.MoveNext() + 0x2d7
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Program.<>c.<<<Main>$>b__0_1>d.MoveNext() + 0x231
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Microsoft.AspNetCore.Http.Generated.<GeneratedRouteBuilderExtensions_g>FB49A76E00958A78ED8A3DC031D4149080B37ED6EB129C512F9112A3DB80F087D__GeneratedRouteBuilderExtensionsCore.<>c__DisplayClass3_0.<<MapPost0>g__RequestHandler|4>d.MoveNext() + 0x3b8
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.<ProcessRequests>d__238`1.MoveNext() + 0x46f
api01    |         Exception data:
api01    |           Severity: FATAL
api01    |           SqlState: 53300
api01    |           MessageText: sorry, too many clients already
api01    |           File: proc.c
api01    |           Line: 351
api01    |           Routine: InitProcess
api01    | fail: Microsoft.AspNetCore.Server.Kestrel[13]
api01    |       Connection id "0HN1AIQR4A40B", Request id "0HN1AIQR4A40B:00000001": An unhandled exception was thrown by the application.
api01    |       Npgsql.PostgresException (0x80004005): 53300: sorry, too many clients already
api01    |          at Npgsql.Internal.NpgsqlConnector.<ReadMessageLong>d__233.MoveNext() + 0x995
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Threading.Tasks.Sources.ManualResetValueTaskSourceCore`1.ThrowForFailedGetResult() + 0x16
api01    |          at System.Threading.Tasks.Sources.ManualResetValueTaskSourceCore`1.GetResult(Int16) + 0x3e
api01    |          at System.Runtime.CompilerServices.PoolingAsyncValueTaskMethodBuilder`1.StateMachineBox`1.System.Threading.Tasks.Sources.IValueTaskSource<TResult>.GetResult(Int16) + 0x2c
api01    |          at Npgsql.Internal.NpgsqlConnector.<Authenticate>d__0.MoveNext() + 0x1f6
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Npgsql.Internal.NpgsqlConnector.<<Open>g__OpenCore|213_1>d.MoveNext() + 0x531
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Npgsql.Internal.NpgsqlConnector.<Open>d__213.MoveNext() + 0x6d9
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Npgsql.PoolingDataSource.<OpenNewConnector>d__37.MoveNext() + 0x54b
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Npgsql.PoolingDataSource.<<Get>g__RentAsync|34_0>d.MoveNext() + 0x1c7
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Npgsql.NpgsqlConnection.<<Open>g__OpenAsync|42_0>d.MoveNext() + 0x3de
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at api.services.ClienteRepository.<CreateTransacao>d__3.MoveNext() + 0x14e
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at api.services.ClientesService.<Transacao>d__3.MoveNext() + 0x2d7
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Program.<>c.<<<Main>$>b__0_1>d.MoveNext() + 0x231
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Microsoft.AspNetCore.Http.Generated.<GeneratedRouteBuilderExtensions_g>FB49A76E00958A78ED8A3DC031D4149080B37ED6EB129C512F9112A3DB80F087D__GeneratedRouteBuilderExtensionsCore.<>c__DisplayClass3_0.<<MapPost0>g__RequestHandler|4>d.MoveNext() + 0x3b8
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.<ProcessRequests>d__238`1.MoveNext() + 0x46f
api01    |         Exception data:
api01    |           Severity: FATAL
api01    |           SqlState: 53300
api01    |           MessageText: sorry, too many clients already
api01    |           File: proc.c
api01    |           Line: 351
api01    |           Routine: InitProcess
api01    | fail: Microsoft.AspNetCore.Server.Kestrel[13]
api01    |       Connection id "0HN1AIQR4A3VG", Request id "0HN1AIQR4A3VG:00000001": An unhandled exception was thrown by the application.
api01    |       Npgsql.PostgresException (0x80004005): 53300: sorry, too many clients already
api01    |          at Npgsql.Internal.NpgsqlConnector.<ReadMessageLong>d__233.MoveNext() + 0x995
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Threading.Tasks.Sources.ManualResetValueTaskSourceCore`1.ThrowForFailedGetResult() + 0x16
api01    |          at System.Threading.Tasks.Sources.ManualResetValueTaskSourceCore`1.GetResult(Int16) + 0x3e
api01    |          at System.Runtime.CompilerServices.PoolingAsyncValueTaskMethodBuilder`1.StateMachineBox`1.System.Threading.Tasks.Sources.IValueTaskSource<TResult>.GetResult(Int16) + 0x2c
api01    |          at Npgsql.Internal.NpgsqlConnector.<Authenticate>d__0.MoveNext() + 0x1f6
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Npgsql.Internal.NpgsqlConnector.<<Open>g__OpenCore|213_1>d.MoveNext() + 0x531
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Npgsql.Internal.NpgsqlConnector.<Open>d__213.MoveNext() + 0x6d9
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Npgsql.PoolingDataSource.<OpenNewConnector>d__37.MoveNext() + 0x54b
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Npgsql.PoolingDataSource.<<Get>g__RentAsync|34_0>d.MoveNext() + 0x1c7
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Npgsql.NpgsqlConnection.<<Open>g__OpenAsync|42_0>d.MoveNext() + 0x3de
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at api.services.ClienteRepository.<CreateTransacao>d__3.MoveNext() + 0x14e
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at api.services.ClientesService.<Transacao>d__3.MoveNext() + 0x2d7
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Program.<>c.<<<Main>$>b__0_1>d.MoveNext() + 0x231
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Microsoft.AspNetCore.Http.Generated.<GeneratedRouteBuilderExtensions_g>FB49A76E00958A78ED8A3DC031D4149080B37ED6EB129C512F9112A3DB80F087D__GeneratedRouteBuilderExtensionsCore.<>c__DisplayClass3_0.<<MapPost0>g__RequestHandler|4>d.MoveNext() + 0x3b8
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.<ProcessRequests>d__238`1.MoveNext() + 0x46f
api01    |         Exception data:
api01    |           Severity: FATAL
api01    |           SqlState: 53300
api01    |           MessageText: sorry, too many clients already
api01    |           File: proc.c
api01    |           Line: 351
api01    |           Routine: InitProcess
api01    | fail: Microsoft.AspNetCore.Server.Kestrel[13]
api01    |       Connection id "0HN1AIQR4A409", Request id "0HN1AIQR4A409:00000001": An unhandled exception was thrown by the application.
api01    |       Npgsql.PostgresException (0x80004005): 53300: sorry, too many clients already
api01    |          at Npgsql.Internal.NpgsqlConnector.<ReadMessageLong>d__233.MoveNext() + 0x995
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Threading.Tasks.Sources.ManualResetValueTaskSourceCore`1.ThrowForFailedGetResult() + 0x16
api01    |          at System.Threading.Tasks.Sources.ManualResetValueTaskSourceCore`1.GetResult(Int16) + 0x3e
api01    |          at System.Runtime.CompilerServices.PoolingAsyncValueTaskMethodBuilder`1.StateMachineBox`1.System.Threading.Tasks.Sources.IValueTaskSource<TResult>.GetResult(Int16) + 0x2c
api01    |          at Npgsql.Internal.NpgsqlConnector.<Authenticate>d__0.MoveNext() + 0x1f6
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Npgsql.Internal.NpgsqlConnector.<<Open>g__OpenCore|213_1>d.MoveNext() + 0x531
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Npgsql.Internal.NpgsqlConnector.<Open>d__213.MoveNext() + 0x6d9
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Npgsql.PoolingDataSource.<OpenNewConnector>d__37.MoveNext() + 0x54b
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Npgsql.PoolingDataSource.<<Get>g__RentAsync|34_0>d.MoveNext() + 0x1c7
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Npgsql.NpgsqlConnection.<<Open>g__OpenAsync|42_0>d.MoveNext() + 0x3de
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at api.services.ClienteRepository.<CreateTransacao>d__3.MoveNext() + 0x14e
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at api.services.ClientesService.<Transacao>d__3.MoveNext() + 0x2d7
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Program.<>c.<<<Main>$>b__0_1>d.MoveNext() + 0x231
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Microsoft.AspNetCore.Http.Generated.<GeneratedRouteBuilderExtensions_g>FB49A76E00958A78ED8A3DC031D4149080B37ED6EB129C512F9112A3DB80F087D__GeneratedRouteBuilderExtensionsCore.<>c__DisplayClass3_0.<<MapPost0>g__RequestHandler|4>d.MoveNext() + 0x3b8
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.<ProcessRequests>d__238`1.MoveNext() + 0x46f
api01    |         Exception data:
api01    |           Severity: FATAL
api01    |           SqlState: 53300
api01    |           MessageText: sorry, too many clients already
api01    |           File: proc.c
api01    |           Line: 351
api01    |           Routine: InitProcess
api01    | fail: Microsoft.AspNetCore.Server.Kestrel[13]
api01    |       Connection id "0HN1AIQR4A3VA", Request id "0HN1AIQR4A3VA:00000001": An unhandled exception was thrown by the application.
api01    |       Npgsql.PostgresException (0x80004005): 53300: sorry, too many clients already
api01    |          at Npgsql.Internal.NpgsqlConnector.<ReadMessageLong>d__233.MoveNext() + 0x995
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Threading.Tasks.Sources.ManualResetValueTaskSourceCore`1.ThrowForFailedGetResult() + 0x16
api01    |          at System.Threading.Tasks.Sources.ManualResetValueTaskSourceCore`1.GetResult(Int16) + 0x3e
api01    |          at System.Runtime.CompilerServices.PoolingAsyncValueTaskMethodBuilder`1.StateMachineBox`1.System.Threading.Tasks.Sources.IValueTaskSource<TResult>.GetResult(Int16) + 0x2c
api01    |          at Npgsql.Internal.NpgsqlConnector.<Authenticate>d__0.MoveNext() + 0x1f6
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Npgsql.Internal.NpgsqlConnector.<<Open>g__OpenCore|213_1>d.MoveNext() + 0x531
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Npgsql.Internal.NpgsqlConnector.<Open>d__213.MoveNext() + 0x6d9
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Npgsql.PoolingDataSource.<OpenNewConnector>d__37.MoveNext() + 0x54b
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Npgsql.PoolingDataSource.<<Get>g__RentAsync|34_0>d.MoveNext() + 0x1c7
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at Npgsql.NpgsqlConnection.<<Open>g__OpenAsync|42_0>d.MoveNext() + 0x3de
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at api.services.ClienteRepository.<CreateTransacao>d__3.MoveNext() + 0x14e
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api01    |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api01    |          at api.services.ClientesService.<Transacao>d__3.MoveNext() + 0x2d7
api01    |       --- End of stack trace from previous location ---
api01    |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
